object-recognition/src/App.js

```diff
@@ -0,0 +1,26 @@
+import logo from './logo.svg';
+import './App.css';
+import HomePage from './components/HomePage';
+import LoginPage from './components/LoginPage';
+import RegisterPage from './components/RegisterPage';
+import FlashcardsPage from './components/FlashcardsPage';
+
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+
+
+function App() {
+    return (
+        <Router>
+            <Routes>
+                <Route path="/" exact component={HomePage} />
+                <Route path="/login" component={LoginPage} />
```

Comment:
For now user have to go to `/login` path for logging in, it's kind of manual stuff, any better ideas here?